Python/Product/PythonTools/ptvsd/debugger.py

```diff
@@ -14,7 +14,7 @@
 # See the Apache Version 2.0 License for specific language governing
 # permissions and limitations under the License.
 
-from __future__ import with_statement
+from __future__ import with_statement, print_function
```

Comment:
Not actually used anywhere - I used it for print-debugging. But it's one less line to change when inserting prints, and most importantly, now that we need 2.6+ anyway, it can be here without any ill effects.